### PR TITLE
Add Cinder PV limit - BZ 1732301

### PR DESCRIPTION
**Pull request by maxwelldb**
Context: https://bugzilla.redhat.com/show_bug.cgi?id=1732301

### CODE REVIEW

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL--small change.

File: install_config/persistent_storage/persistent_storage_cinder.adoc [hunk 1]
```diff
@@ -190,3 +190,27 @@ spec:
...
+. Set the `KUBE_MAX_PD_VOLS` environment variable to an integer. For example, in `/etc/origin/master/master.env`:
++
+----
+KUBE_MAX_PD_VOLS=26
+----
+
+. From a command line, restart the API service:
++
+----
+$ master-restart api
```
Comment (bobfuru) [inline install_config/persistent_storage/persistent_storage_cinder.adoc:209]: A quick check of 3.11 docs shows `master-restart api` and `master-restart controllers` (below) to be executed from root. For example: https://docs.openshift.com/container-platform/3.11/install_config/configuring_vsphere.html#vsphere-applying-configuration-changes

So maybe `$` -> `#` ?

**Review by bobfuru — COMMENTED** [review]
One question and otherwise LGTM.

File: install_config/persistent_storage/persistent_storage_cinder.adoc (hunk 1 above)
Comment (maxwelldb) [reply install_config/persistent_storage/persistent_storage_cinder.adoc:209]: @bobfuru Fair. 

@chao007 Could you comment on this? The change would warrant a QE re-check if it happened, anyway. :)

**Comment by chao007** [comment]
/lgtm
Thanks

**Comment by maxwelldb** [comment]
@chao007 To be clear: we should leave the PR as it is, or should the commands in https://github.com/openshift/openshift-docs/pull/22220#discussion_r450939070 be executed as root?

**Comment by maxwelldb** [comment]
@chao007 Any updates on this?

File: install_config/persistent_storage/persistent_storage_cinder.adoc (hunk 1 above)
Comment (chao007) [reply install_config/persistent_storage/persistent_storage_cinder.adoc:209]: Yes. We should update to `#` here

Comment (maxwelldb) [reply install_config/persistent_storage/persistent_storage_cinder.adoc:209]: @chao007 Thanks!

**Pull request merged**